lightning/checkpoints/checkpoints.go

```diff
@@ -388,15 +389,13 @@ func NewMySQLCheckpointsDB(ctx context.Context, db *sql.DB, schemaName string) (
 
 	err = sql.Exec(ctx, "create table checkpoints table", fmt.Sprintf(`
 		CREATE TABLE IF NOT EXISTS %s.%s (
-			node_id int unsigned NOT NULL,
-			session bigint unsigned NOT NULL,
+			task_id bigint NOT NULL,
```

Comment:
can we combine `task_id` and `table_name` as a UNIQUE KEY or is it needed?

Reply:
Not strictly needed, this is mainly for debugging currently.